includes/class-wc-data-structurer.php

```diff
@@ -0,0 +1,180 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+/**
+ * Structured data's handler and generator using JSON-LD format.
+ *
+ * @class 		WC_Data_Structurer
+ * @version		2.7.0
+ * @package		WooCommerce/Classes
+ * @category	Class
+ * @author 		Clement Cazaud
+ */
+class WC_Data_Structurer {
```

Comment:
WC_Structured_Data?

Not to be pedantic, but I don't want this to be confused with our CRUD data classes.

@claudiosmweb thoughts?